File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        
         PriorityQueue<Integer> Maxheap = new PriorityQueue<>();

         for(int num : nums){

              Maxheap.add(num);

              if(Maxheap.size() > k){

                     Maxheap.poll();
              }
         }
            return Maxheap.peek();
    }
}

File: 0215-kth-largest-element-in-an-array/0215-kth-largest-element-in-an-array.cpp
class Solution {
public:
    int findKthLargest(vector<int>& nums, int k) {
        
        priority_queue<int , vector<int>, greater<int>>maxheap;

        for(int num : nums){

                maxheap.push(num);

                if(maxheap.size() > k){

                      maxheap.pop();
                }
        }
             return maxheap.top();
    }
};